R/tweak-page.R

```diff
@@ -54,6 +54,12 @@ tweak_rmarkdown_html <- function(html, input_path, pkg = pkg) {
     )
   }
 
+  # If top-level headings use h1, move eveything down one level
+  h1 <- xml2::xml_find_all(html, "//h1")
+  if (length(h1) > 1) {
+    tweak_section_levels(html)
```

Comment:
Actually the code should not move _everything_ down one level, only the non page-header header. 
When `tweak_section_levels()` is called for news it is only called on the content of NEWS.md i.e. without any page header.